core/domain/email_manager_test.py

```diff
@@ -5332,6 +5332,38 @@ def test_that_email_not_sent_if_can_send_emails_is_false(self):
             self.APPLICANT_EMAIL)
         self.assertEqual(len(messages), 0)
 
+    def test_account_deletion_failed_email_is_sent_correctly(self):
+        dummy_admin_address = 'admin@system.com'
+
+        admin_email_ctx = self.swap(
+            feconf, 'ADMIN_EMAIL_ADDRESS', dummy_admin_address)
+
+        with self.can_send_emails_ctx, admin_email_ctx:
+            # Make sure there are no emails already sent.
+            messages = self._get_sent_email_messages(
+                feconf.ADMIN_EMAIL_ADDRESS)
+            self.assertEqual(len(messages), 0)
```

Comment:
I think it'd be clearer to have:

```suggestion
            self.assertEqual(messages, [])
```

Reply:
Done